### ISSUE TITLE
Account destroy fails on stable.dev

### ISSUE THREAD

**Issue by vladikoff** [mozilla/fxa#2265]
Ref: https://circleci.com/gh/mozilla/application-services/27330?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
ntegration-test-helper:     Response body: {"code":500,"errno":999,"error":"Internal Server Error","message":"An internal server error occurred","info":"https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/docs/api.md#response-format","request":{"locale":"en","userAgent":{"browserVersion":null,"osVersion":null,"deviceType":null},"method":"post","path":"/auth/v1/account/destroy","query":{},"payload":{},"headers":{"x-forwarded-for":"34.228.26.205, 172.31.14.103","host":"stable.dev.lcip.org","connection":"close","content-length":"127","x-forwarded-proto":"https","x-forwarded-port":"443","x-amzn-trace-id":"Root=1-5d5b4181-a420601f1af0d456f67abb3d","content-type":"application/json","accept-encoding":"gzip,deflate","user-agent":"node-fetch/1.0 (+https://github.com/bitinn/node-fetch)","accept":"*/*"}}}
integration-test-helper: Error: 
```

cc @jrgm @jbuck

**Comment by vladikoff** [comment]
We unblocked our tests, but it seems like if account is deleted too early it might cause 500s

**Comment by eoger** [comment]
Duplicate of https://github.com/mozilla/fxa/issues/3629.
